src/ops/upgrade/mod.rs

```diff
@@ -64,11 +64,18 @@ pub fn main(upgrade_target: cli::UpgradeTo) -> OpResult {
     let tmpdir = tempdir().unwrap();
     match expr.clone().attribute("package").path(&tmpdir.path()) {
         Ok(build_result) => {
-            let ret = Command::new("nix-env")
+            let status = Command::new("nix-env")
                 .arg("--install")
                 .arg(build_result)
-                .status();
-            Ok(Some(format!("{:?}", ret)))
+                .status() 
+                .expect("Error: failed to execute nix-env --install");
```

Comment:
The `expect` here means lorri will blow up in the users face when `nix-env` fails to execute but I suppose in that case exploding is probably justified.